### PR TITLE
 Issue #SB-24156 fix: Fixed the hex code displayed issue

### PR DESCRIPTION
**Pull request by christyfernandes**

### CODE REVIEW

File: projects/discussion-ui/src/lib/components/discuss-start/discuss-start.component.ts [hunk 1]
```diff
@@ -67,13 +67,18 @@ export class DiscussStartComponent implements OnInit {
       const tags = _.map(_.get(topicData, 'tags'), (element) => {
         return _.get(element, 'value');
       });
-      this.startForm.controls['question'].setValue(_.get(topicData, 'title'));
+      this.startForm.controls['question'].setValue(this.htmlDecode(_.get(topicData, 'title')));
       this.startForm.controls['description'].setValue(_.get(topicData, 'posts[0].content').replace(/<[^>]+>/g, ''));
       this.startForm.controls['tags'].setValue(tags);
       this.validateForm();
     }
   }
```
Comment (sudip-mukherjee) [inline projects/discussion-ui/src/lib/components/discuss-start/discuss-start.component.ts:76]: Add functional comment for this method: htmlDecode(str)

**Review by sudip-mukherjee — CHANGES_REQUESTED** [review]

File: projects/discussion-ui/src/lib/components/discuss-start/discuss-start.component.ts [hunk 2]
```diff
@@ -67,13 +67,19 @@ export class DiscussStartComponent implements OnInit {
...
 
+  /** The htmlDecode() method parses a string containing either HTML or XML */
```
Comment (vinukumar-vs) [inline projects/discussion-ui/src/lib/components/discuss-start/discuss-start.component.ts:77]: JSDOC format is not correct.

Is this method only for this component. Can't this method can be used by other components? 
If other components can use, then it should be in util service.

Comment (vinukumar-vs) [reply projects/discussion-ui/src/lib/components/discuss-start/discuss-start.component.ts:77]: This follows the
 SRP principle -> This is not related to this component.
 DRY principle -> If other components requires the same method they have to copy-paste the same in another component
 
 @christyfernandes @sudip-mukherjee 

This mehtod is

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: projects/discussion-ui/src/lib/services/discuss-utils.service.ts [hunk 3]
```diff
@@ -24,4 +24,13 @@ export class DiscussUtilsService {
     return 'rgba(255, 255, 255, 80%)';
   }
 
+  /**
+   * The htmlDecode() method parses a string containing either HTML or XML
+   * @param  {any} str
```
Comment (vinukumar-vs) [inline projects/discussion-ui/src/lib/services/discuss-utils.service.ts:29]: Start adding the description for the parameters as well
Add example also. It helps to understand the code better.

**Review by vinukumar-vs — APPROVED** [review]

**Pull request merged**